src/universal/validation/makeProjectSchema.js

```diff
@@ -1,17 +1,35 @@
+import {ContentState, convertToRaw} from 'draft-js';
+import {columnArray, PROJECT_MAX_CHARS} from 'universal/utils/constants';
 import {compositeId} from 'universal/validation/templates';
-import {columnArray} from 'universal/utils/constants';
 import legitify from './legitify';
-import {PROJECT_MAX_CHARS} from 'universal/utils/constants';
+
+const makeEmptyStr = () => JSON.stringify(convertToRaw(ContentState.createFromText('')));
 
 export default function makeProjectSchema() {
   return legitify({
     id: compositeId,
     agendaId: compositeId,
     content: (value) => value
       .trim()
+      .normalize((str) => {
+        let parsedContent;
+        try {
+          parsedContent = JSON.parse(str);
+        } catch (e) {
+          return makeEmptyStr();
+        }
+        const keys = Object.keys(parsedContent);
+        if (keys.length !== 2 ||
+          typeof parsedContent.entityMap !== 'object' ||
+          !Array.isArray(parsedContent.blocks ||
+          parsedContent.blocks.length === 0)) {
+          return makeEmptyStr();
+        }
+        return str;
+      })
       .max(PROJECT_MAX_CHARS, 'Whoa! That looks like 2 projects'),
```

Comment:
You know, somehow I never saw that validation error string before. Heh. I like it.